packages/svelte/src/compiler/utils/fuzzymatch.js

```diff
@@ -19,18 +19,20 @@ const GRAM_SIZE_UPPER = 3;
  * @param {string} str2
  */
 function _distance(str1, str2) {
-	if (str1 === null && str2 === null) {
+	if (!str1 && !str2) {
```

Comment:
This changes the behavior of the code. What about empty string? Are you sure this is the right thing to do?